pkg/api/converterfromapi.go

```diff
@@ -712,7 +255,6 @@ func convertDcosConfigToVLabs(api *DcosConfig, vl *vlabs.DcosConfig) {
 
 func convertKubernetesConfigToVLabs(apiCfg *KubernetesConfig, vlabsCfg *vlabs.KubernetesConfig) {
 	vlabsCfg.KubernetesImageBase = apiCfg.KubernetesImageBase
-	vlabsCfg.MCRKubernetesImageBase = apiCfg.MCRKubernetesImageBase
```

Comment:
Do we want this change?

Reply:
hm... where did that come from, I'll revert that line, thanks 🦅

Reply:
fixed and squashed into the offending commit